examples/basic.html

```diff
@@ -38,6 +38,10 @@
           new CustomEvent('WebComponentsReady', {bubbles: true})
         );
       }
+
+      window.addEventListener('px-accordion-content-icon-clicked', (e) => {
```

Comment:
Can you remove this?

Reply:
Hi @evanjd I have remove the above and rebased. The commit is now under my authorised email address joeburton@gmail.com. Is there anything else I need to do?

Reply:
You'll need to rebase to fix your other two commits which still have an invalid author. I can do this for you if you want.